continuous_integration/scripts/build-legate-conda

```diff
@@ -58,8 +61,15 @@ cmake_version:
   - ">=3.20.1,!=3.23.0"
 package_version:
   - "$(git -C $REPO_DIR describe --abbrev=0 --tags | $SED 's/[a-zA-Z]//g' | cut -d '.' -f -2).00"
+ucx_configured:
+  - "${UCX_CONFIGURED}"
 EOF
-
+if [ "$UCX_ENABLED" = "ON" ]; then
+    cat <<EOF >> $REPO_DIR/conda/conda-build/conda_build_config.yaml
+ucx:
+  - ">=1.14"
```

Comment:
>=1.14? is there a reason we pick 1.14 specifically?

Reply:
As per mentioned here: https://github.com/nv-legate/legate.core/blob/branch-24.01/BUILD.md

Reply:
ah, makes sense, then that number should probably also be reflected in the meta.yaml as well? (I'll make a separate comment there)